web/components/ui/Header/Header.tsx

```diff
@@ -57,9 +59,10 @@ export const Header: FC<HeaderComponentProps> = ({ name, chatAvailable, chatDisa
         <UserDropdown id="user-menu" hideTitleOnMobile showToggleChatOption={canHideChat} />
       )}
       {!chatAvailable && !chatDisabled && (
-        <Tooltip
+        
+       <Tooltip
```

Comment:
I'm thinking we shouldn't show any of this if chat is disabled. No tooltip, no text, no anything.